genie-core/src/test/java/com/netflix/genie/core/jpa/entities/ClusterEntityUnitTests.java

```diff
@@ -81,6 +79,8 @@ public void testDefaultConstructor() {
         Assert.assertNull(entity.getVersion());
```

Comment:
As you can probably tell looking at the code a lot of these tests were developed at various points of knowledge for the best way to test things. Basically this is kind of the worst where you have Junit + Mockito + raw Assert statements. Next best would be how it is in some other places where it uses hamcrest Matchers cause the error messages are better and it gives us more flexibility. The best (imo) is probably the Spock stuff as it removes a lot of the boiler plate and can be more readable. More technical debt we might want to address at some point. Outside the scope of this PR just pointing it out as general comment going forward you might want to develop new tests using Spock.